cmd/srcd/cmd/config.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/src-d/engine/cmd/srcd/config"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// configCmd represents the config command
+type configCmd struct {
+	Command `name:"config" short-description:"Edit the config file" long-description:"Edit the config file.\n\nIf the config file is empty it will be populated with the default values.\nSet EDITOR env var to change the text editor command."`
+}
+
+func (c *configCmd) Execute(args []string) error {
+	cmdName := os.Getenv("VISUAL")
+	if cmdName == "" {
+		cmdName = os.Getenv("EDITOR")
+	}
+
+	if cmdName == "" {
+		if runtime.GOOS == "windows" {
+			cmdName = "start" // opens the default editor
+		} else {
+			cmdName = "nano"
+		}
+
+		log.Debugf("EDITOR is not set, using command %s", cmdName)
+	}
+
+	configFile := c.Config
+	if configFile == "" {
+		var err error
+		configFile, err = config.DefaultPath()
+		if err != nil {
+			return err
+		}
+	}
+
+	var emptyFile bool
+
+	_, err := os.Stat(configFile)
```

Comment:
nit: I'd move all this check into a `func isEmptyConfig(path string) (bool, error)` instead of only separating part of the logic into the `fileIsEmpty` function.

Reply:
Done in 1c2cf99